docs/docs/getting-started/mini-constellation.md

```diff
@@ -0,0 +1,115 @@
+# Mini Constellation
+
+With `constellation mini`, users can quickly deploy and test Constellation without the need for a cloud subscription.
+
+The command uses virtualization to create a local cluster with one control-plane and one worker node.
+
+:::info
+
+Since mini Constellation is running on your local system, please note that common cloud features, such as load-balancing,
+attaching persistent storage, or auto-scaling, are unavailable.
+
+:::
+
+## Prerequisites
+
+* [Constellation CLI](./install.md#install-the-constellation-cli)
+* Linux operating system on amd64 hardware
+* At least 6GB RAM
+* A CPU with at least 4 cores
```

Comment:
```suggestion
* A x86-64 CPU with at least 4 cores
* Hardware virtualization enabled in the BIOS/UEFI (often referred to as Intel VT-x or AMD-V/SVM)
```
Unfortunately many HW vendors did disable virtualization in the BIOS/UEFI by default for many years so I would point that out.

Maybe I would also move this to the top of the list.